CalculateTotal.js

```diff
@@ -0,0 +1,42 @@
+function getOrderTotal(items) {
+  var total = 0;
+  items.map((x) => {
+    total += x.cost * x.quantity;
+  });
+
+  // console.log("total", total);
+  return total;
+}
+
+var calculateSalesTax = function(total) {
+  return total + (total * 15) / 100;
+};
+
+function calculateTotal(itemsInCart) {
+  var merchandiseTotal = getMerchandiseTotal(itemsInCart);
+  var shippingCost
+  if (merchandiseTotal > 35) shippingCost = 0
+  else {
+    shippingCost = 10;
+  }
```

Comment:
There is inconsistent use of braces here. It is good practice to follow a consistent style.

Also, this block of code can be converted into its own function to calculate the shipping cost. 

Example: 

```js
function getShippingCost(total) {
  return total > 35 ? 0 : 10;
}
```

So, the `calculateTotal` function will look like: 

```js
function calculateTotal(itemsInCart) {
  const merchandiseTotal = getOrderTotal(itemsInCart);
  const shippingCost = getShippingCost(merchandiseTotal);
  const salesTax = calculateSalesTax(merchandiseTotal);

  return merchandiseTotal + shippingCost + salesTax;
}
```